dotnet-testing/MockingWithNSubstitute/MockingWithNSubstitute.Tests/NotificationServiceTests.cs

```diff
@@ -0,0 +1,72 @@
+﻿using FluentAssertions;
+using NSubstitute;
+
+namespace MockingWithNSubstitute.Tests
```

Comment:
We can use file scoped namespace to reduce nesting

Reply:
Sorted